dag-json/src/codec.rs

```diff
@@ -39,14 +39,15 @@ fn serialize<S: ser::Serializer>(ipld: &Ipld, ser: S) -> Result<S::Ok, S::Error>
             ser.collect_map(wrapped)
         }
         #[cfg(feature = "unleashed")]
-        Ipld::IntegerMap(_) => {
-            // TODO: how to return an error here?
-            unimplemented!()
+        Ipld::IntegerMap(map) => {
+            let wrapped = map.iter().map(|(key, ipld)| (key, Wrapper(ipld)));
+            ser.collect_map(wrapped)
```

Comment:
So this will produce json like ```{ 1: "foo", 2: "bar" }```?

Reply:
This lets serde_json take care of it, I think this will result in string keys

Reply:
serde_json will produce int keys for an int keyed map as far as I know. But that is what I want, so it is fine.

Reply:
The keys will likely be a string as JSON doesn't support numbers as keys for objects (only JavaScript does).

Reply:
Right. Forgot about that. Anyway, this just to have a way to produce debug output for arbitrary dag-cbor unleashed data, so it is fine.